### PR TITLE
Astro 2.0

### PR DESCRIPTION
**Pull request by Maarten-Dekker**
This PR upgrades Astroship to Astro 2.0.

I know this is not really the place for new features, but I made the following changes:

- Blog & Team will only be displayed when the date is not in the future.
- Added the ability to mark blog posts and team members as drafts.
- The word "excerpt" has been changed to "snippet".
- Removed the blog post layout attribute.

Feel free to change this PR to your liking.

### CODE REVIEW

**Comment by surjithctly** [comment]
Hey,

Thanks so much for working on this. Looks good to me. Added pnpm lock and deployed the preview. 

Is it all good, I can merge it.

**Comment by Maarten-Dekker** [comment]
Thank you for this nice, lean theme. Ready to merge.

**Review by surjithctly — APPROVED** [review]

File: astro.config.mjs [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import { defineConfig } from "astro/config";
 import tailwind from "@astrojs/tailwind";
 import image from "@astrojs/image";
 import mdx from "@astrojs/mdx";
 import sitemap from "@astrojs/sitemap";
 
-export default defineConfig({
```
[the remark this answers is not in the document]
Comment (Maarten-Dekker) [reply astro.config.mjs]: Woops, shouldn't have been in the request. Thanks for fixing.

**Comment by surjithctly** [comment]
Thanks for the PR Man! really appreciate it! 🙏

**Pull request merged**